shell/job.c

```diff
@@ -226,7 +226,11 @@ static bool _job_wait(struct mrsh_state *state, pid_t pid, int options) {
 	// shell. Child processes want to block until their own children have
 	// terminated.
 	if (!priv->child) {
+#ifdef WCONTINUED
```

Comment:
Without this the Archlinux build fails which is interesting since this was introduced in Linux afaik. It could be because of the build options though if they are excluding this somehow by specifying an older standard.